File: KnowledgeSpaceApk/adminapp/src/main/java/com/example/adminapp/FeesActAdm/FeesResultFragFeesActAdm.java
package com.example.adminapp.FeesActAdm;

import android.os.Bundle;

import androidx.fragment.app.Fragment;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.example.adminapp.R;

import java.util.ArrayList;


public class FeesResultFragFeesActAdm extends Fragment {

    private ArrayList<DataModelRecVFeesResultFAdm> dataHolder;
    private AdapterFeesResultFAdm adapter;
    private RecyclerView recyclerVFeesResultFAdm;

    public FeesResultFragFeesActAdm() {
        // Required empty public constructor
    }


    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        View view = LayoutInflater.from(getContext()).inflate(R.layout.fragment_fees_result_frag_fees_act_adm, container, false);
        recyclerVFeesResultFAdm = view.findViewById(R.id.recyclerVFeesResultFAdm);
        recyclerVFeesResultFAdm.setLayoutManager(new LinearLayoutManager(getContext()));
        addDataToHolder();
        recyclerVFeesResultFAdm.setAdapter(new AdapterFeesResultFAdm(dataHolder));


        return view;
    }

    private void addDataToHolder() {
        dataHolder = new ArrayList<>();
        DataModelRecVFeesResultFAdm obj = new DataModelRecVFeesResultFAdm(452680,"Shrikant",
                56000);
        dataHolder.add(obj);
    }
}

File: KnowledgeSpaceApk/adminapp/src/main/java/com/example/adminapp/FeesActAdm/FeesResultFragFeesActAdm.kt
package com.example.adminapp.FeesActAdm

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView
import com.example.adminapp.R

class FeesResultFragFeesActAdm : Fragment() {
    private var dataHolder: ArrayList<DataModelRecVFeesResultFAdm>? = null
    private val adapter: AdapterFeesResultFAdm? = null
    private var recyclerVFeesResultFAdm: RecyclerView? = null
    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        // Inflate the layout for this fragment
        val view = LayoutInflater.from(context)
            .inflate(R.layout.fragment_fees_result_frag_fees_act_adm, container, false)
        recyclerVFeesResultFAdm = view.findViewById(R.id.recyclerVFeesResultFAdm)
        recyclerVFeesResultFAdm?.setLayoutManager(LinearLayoutManager(context))
        addDataToHolder()
        recyclerVFeesResultFAdm?.setAdapter(AdapterFeesResultFAdm(dataHolder))
        return view
    }

    private fun addDataToHolder() {
        dataHolder = ArrayList()
        val obj = DataModelRecVFeesResultFAdm(
            452680, "Shrikant",
            56000
        )
        dataHolder!!.add(obj)
    }
}